magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/services/S3Discovery.java

```diff
@@ -42,11 +42,13 @@
 import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
 
+import static io.openraven.magpie.plugins.aws.discovery.AWSUtils.apply;
 import static io.openraven.magpie.plugins.aws.discovery.AWSUtils.getAwsResponse;
 
 public class S3Discovery implements AWSDiscovery {
 
   private static final String SERVICE = "s3";
+  private final String RESOURCE_TYPE = "AWS::S3::Bucket";
```

Comment:
Why did you move this to class member?

Reply:
Since such Strings is rather a discovery class related constant and used as is, I propose to not create it each time during invocation of the discovery. If will be agreed to do so that will be made in separate PR for all discoveries

Reply:
Defining RESOURCE_TYPE in function seems clearer to me.
For sure in approach proposed by you RESOURCE_TYPE have to be renamed to something else like RESOURCE_TYPE_S3_BUCKET. This is because there are cases with multiple RESOURCE_TYPEs in one class

Reply:
I will take off this change so far